cppForSwig/cryptopp/rng.cpp

```diff
@@ -59,25 +59,33 @@ void LC_RNG::GenerateBlock(byte *output, size_t size)
 #ifndef CRYPTOPP_IMPORTS
 
 X917RNG::X917RNG(BlockTransformation *c, const byte *seed, const byte *deterministicTimeVector)
-	: cipher(c),
-	  S(cipher->BlockSize()),
-	  dtbuf(S),
-	  randseed(seed, S),
-	  m_lastBlock(S),
-	  m_deterministicTimeVector(deterministicTimeVector, deterministicTimeVector ? S : 0)
+	: m_cipher(c),
```

Comment:
@goatpig will need to comment on whether or not we want to touch Crypto++. It's not used by default anymore.

Reply:
This was from #312.